ftest.sh

```diff
@@ -312,7 +312,13 @@ if ${SETUP_ONLY:-false}; then
 fi
```

Comment:
The 1st line of the patch still does not match our standard. It is missing a component.

Reply:
DO you mean the title of the commit message that has component (VOS?) in it?

Reply:
This is the commit msg from your origin commit:
DAOS-2786 api: add python module and objclass import

is api the component should be added after the jira number?

Reply:
added component api to the patch.